File: Festival/src/main/java/Validation/ValidatorUser.java
package Validation;

import Domain.User;
import Validation.Exceptions.ValidatorException;

/**
 * Created by Sebi on 18-Mar-17.
 */
public class ValidatorUser implements IValidator<User> {
    @Override
    public void validate(User user) throws ValidatorException {
        String errors = "";
        if (user.getUsername().length() == 0){
            errors += "Username must not be empty";
        }
        if (user.getPassword().length() == 0){
            errors += "Password must not be empty";
        }
        if (errors.length() != 0){
            throw new ValidatorException(errors);
        }
    }
}

File: FestivalC#/Festival/Services/Validation/ValidatorUser.cs
using Festival.Model;
using Festival.Validation.Exceptions;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Festival.Validation
{
    public class ValidatorUser : IValidator<User>
    {
        public void validate(User user)
        {
            String errors = "";
            if (user.Username.Length == 0)
            {
                errors += "Username must not be empty";
            }
            if (user.Password.Length == 0)
            {
                errors += "Password must not be empty";
            }
            if (errors.Length != 0)
            {
                throw new ValidatorException(errors);
            }
        }
    }
}
